lib/calc.rb

```diff
@@ -1,3 +1,33 @@
 # frozen_string_literal: true
 
 # Calculator Class here
+
+class Calculator
+  def initialize(num1)
+    @num1 = num1
```

Comment:
Converting `num1` to a `Float` here will cause all operations to return `Float`s.